csrankings-o.csv

```diff
@@ -102,6 +102,7 @@ Oliver Yang,University of Ottawa,http://www.site.uottawa.ca/~yang,NOSCHOLARPAGE
 Oliver van Kaick,Carleton University,http://people.scs.carleton.ca/~olivervankaick,IfvyBBUAAAAJ
 Olivier Alata,Université Jean Monnet,http://perso.univ-st-etienne.fr/ao29170h,ClqgTp8AAAAJ
 Olivier Caron,CRIStAL,https://www.cristal.univ-lille.fr/profil/carono,NOSCHOLARPAGE
+Olivier Danvy,Ysle NUS,https://www.yale-nus.edu.sg/faculty/olivier-danvy/,OZJGoVoAAAAJ

```

Comment:
This institution (which incidentally is misspelled) is not in the database - faculty should be listed under NUS.